README.md

```diff
@@ -9,18 +9,26 @@ This repository holds the source for this Specification, part of the family of [
 
 ### What does it do?
 
-Describes the specification for the NMOS Configuration API.
+Enables the configuration of an NMOS Nodes' [MS-05-02](https://specs.amwa.tv/ms-05-02/) model using a backup and restore API.
 
 ### Why does it matter?
 
-Sets out the rules and requirements for implementing the NMOS Device Configuration API in devices and controllers.
+Defines a standard way of backing up and restoring [MS-05-02](https://specs.amwa.tv/ms-05-02/) models, as well as defining a way to manage dynamically configurable Device Models.
 
 ### How does it work?
 
-Describes a standard way of exposing [MS-05-02](https://specs.amwa.tv/ms-05-02/) models when using an HTTP API.
+- It specifies an HTTP API for backing up an NMOS Nodes' [MS-05-02](https://specs.amwa.tv/ms-05-02/) model as a backup dataset (JSON description).
```

Comment:
I think we should avoid the term description here so we avoid any confusions where someone might think the backup dataset only includes class and datatype descriptors.
```suggestion
- It specifies an HTTP API for backing up an NMOS Nodes' [MS-05-02](https://specs.amwa.tv/ms-05-02/) model as a JSON backup dataset.
```